File: 프로그래머스/0/181852. 뒤에서 5등 위로/뒤에서 5등 위로.js
function solution(num_list) {
    const answer = new Array(num_list.length - 5);
    // answer의 길이는 num_list에서 제외하고자 하는 수만큼 짧게 선언한다
    num_list.sort((a, b) => a - b);
    // num_list를 정렬한다
    let num = 0;

    for (let i = 0; i < num_list.length; i++) {
        if (i > 4) {
            answer[num] = num_list[i];
            num++;
        } // i가 num_list의 원소 번호가 5 이상일때 answer에 넣어준다
    }
    return answer;
}

File: 프로그래머스/unrated/181852. 뒤에서 5등 위로/뒤에서 5등 위로.java
import java.util.Arrays;

class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[num_list.length - 5];
        // answer의 길이는 num_list에서 제외하고자 하는 수만큼 짧게 선언한다
        Arrays.sort(num_list);
        // num_list를 정렬한다
        int num = 0;
        
        for(int i = 0; i < num_list.length; i++) {
			if(i > 4) {
				answer[num] = num_list[i];
				num++;
			} // i가 num_list의 원소 번호가 5 이상일때 answer에 넣어준다
		}
        return answer;
    }
}